src/sentry/api/endpoints/integration_installation.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import absolute_import
+
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+
+from sentry import roles
+from sentry.api.base import Endpoint, SessionAuthentication
+from sentry.api.serializers import serialize
+from sentry.api.serializers.models.organization import DetailedOrganizationSerializer
+from sentry.models import Organization, OrganizationMember, OrganizationStatus
+
+
+class IntegrationInstallationEndpoint(Endpoint):
+    authentication_classes = (SessionAuthentication, )
+    permission_classes = (IsAuthenticated, )
+
+    def get(self, request):
```

Comment:
you can use the org index endpoint and filter to orgs that the user owns: https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/organization_index.py#L72

Reply:
👍 one less endpoint!